deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -0,0 +1,752 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  - Creates a CodeBuild Project to pull the manifest pipeline GitHub repository,
+  build the lambdas and StepFunctions and package them for deployment.
+  - Creates a CodePipeline that has a specific CodeBuild associated,
+  which creates a changed set from the package and deploys it to test and production.
+  - Requires the iiif-service pipeline to be built.
+
+Parameters:
+  AppName:
```

Comment:
@jonhartzler and I discussed this a bit, but noting it here. In general, when connecting stack exports to imports, or SSM parameter paths, you have to follow some convention. I've been sticking with a more generic stack-name approach, instead of breaking out the account-appname-env to lessen how much we impose institutional standards on others, and just to simplify some of the template code.  Our stack names generally incorporate these values in it so it still accomplishes our institutional needs, while allowing for others to name things the way they choose and still have the templates work. 

Looking at how this param is used in this template vs other pipeline templates in this repo, it looks like this should just allow the user to specify the stack name for the test and prod stack that will be created/updated by the pipeline.

Reply:
I would like to handle this a future enhancement because of its size.